packages/backend/src/config/makeConfig.ts

```diff
@@ -128,6 +128,7 @@ export function makeConfig(
       cache: {
         tvl: flags.isEnabled('cache', 'tvl'),
         liveness: flags.isEnabled('cache', 'liveness'),
+        verifiers: flags.isEnabled('cache', 'verifiers'),
```

Comment:
do we need it?

Reply:
yes:
- it's quite slow
- we don't want to query this api to often